### PR TITLE
New user story2

### PR DESCRIPTION
**Pull request by minatimar**

### CODE REVIEW

File: components/DonationListItem.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
+import styled from "styled-components";
+import { useState } from "react";
+
+export default function DonationListItem({ donation }) {
+  const [toggle, setToggle] = useState(false);
+  function handleClick() {
+    setToggle(!toggle);
+  }
+  <h2> Spenden in deiner Nähe: </h2>;
```
Comment (F-Kirchhoff) [inline components/DonationListItem.js]: you don't return the h2 here, that will not be displayed in your app.

File: components/DonationListItem.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    return (
+      <StyledCard>
+        <p>{donation.image}</p>
+        <p>Beschreibung: {donation.description}</p>
+        <p>Entfernung: {donation.distance}</p>
+        <StyledMoreInfoButton type="button" onClick={() => handleClick()}>
+          Mehr Infos
+        </StyledMoreInfoButton>
+      </StyledCard>
+    );
```
Comment (F-Kirchhoff) [inline components/DonationListItem.js:65]: I like your approach a lot with the local state. Your conditional rendering is fine, but be aware that you could solve it by using the `&&` operator. If you don't know how to do that I would recommend you to research it. But again, very nice solution here!

File: pages/_app.js [hunk 3]
```diff
@@ -20,6 +20,12 @@ export default function App({ Component, pageProps }) {
         donations={donations}
         createDonation={createDonation}
       />
+      <Head>
+        <title>Essensretter</title>
+        <meta name="description" content="Generated by create next app" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
```
Comment (F-Kirchhoff) [inline pages/_app.js]: normally its better to put the Head component first, since it is more common (it mimics the HTML file structure)

**Review by F-Kirchhoff — CHANGES_REQUESTED** [review]
Wow, I really like your code. There are two tiny issues, but that won't take long. Great work!

File: components/DonationListItem.js (hunk 1 above)
Comment (minatimar) [reply components/DonationListItem.js]: oh yes forgot to delete it here. its returned from the index.js already

File: components/DonationListItem.js (hunk 2 above)
Comment (minatimar) [reply components/DonationListItem.js:65]: ok great thank you - i will consider the &&-operator in my next component.

File: pages/_app.js (hunk 3 above)
Comment (minatimar) [reply pages/_app.js]: ok thanks - i changed it :)

**Comment by Mistake91** [comment]
hey Mina, i see you did the changes from Felix comments, i guess its up to you to use the && operator, so for me it looks good :)

**Review by Mistake91 — APPROVED** [review]

**Pull request merged**